src/meta/types.ts

```diff
@@ -457,8 +457,10 @@ export interface OpenableOptions {
 	aliases: string[];
 }
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface OpenableOpenOptions {}
+export interface OpenableOpenOptions {
+	lvl?: number;
```

Comment:
name this the skill level since in the future something might need two different skills to calculate its rewards